includes/calendars/views/default-calendar-grid.php

```
@@ -307,16 +307,25 @@ class="simcal-nav simcal-current"
+			if ($a->calendar_priority != $b->calendar_priority) {
+				return $a->calendar_priority < $b->calendar_priority ? -1 : 1;
+			}
 			if ($a->title == $b->title) {
 				return 0;
 			}
 			return $a->title < $b->title ? -1 : 1;
-		} else {
-			return $a->start < $b->start ? -1 : 1;
 		}
+		return $a_local < $b_local ? -1 : 1;
 	}
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'cmp'. (undefined)

(UnusedPrivateMethod)